smoke-tests/apps/JmxMetric/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/JmxMetricTest.java

```diff
@@ -175,7 +175,7 @@ private void verifyJmxMetricsSentToBreeze() throws Exception {
     // versions
     if (testing.getCurrentEnvironment() == TOMCAT_8_JAVA_21
         || testing.getCurrentEnvironment() == TOMCAT_8_JAVA_23) {
-      assertThat(wildcardValueSum).isEqualTo(gcFirstMatch + gcSecondMatch + 6);
+      assertThat(wildcardValueSum).isGreaterThanOrEqualTo(gcFirstMatch + gcSecondMatch);
```

Comment:
`6` here represents the number of other GC collections, but doesn't appears to be quite a fixed number, so relaxing the condition